calendar-bundle/src/Resources/contao/dca/tl_calendar_events.php

```diff
@@ -344,7 +344,7 @@
 			'label'                   => &$GLOBALS['TL_LANG']['tl_content']['singleSRC'],
 			'exclude'                 => true,
 			'inputType'               => 'fileTree',
-			'eval'                    => array('filesOnly'=>true, 'fieldType'=>'radio', 'extensions'=>Config::get('validImageTypes'), 'mandatory'=>true),
+			'eval'                    => array('filesOnly'=>true, 'fieldType'=>'radio', 'extensions'=>implode(',', System::getContainer()->getParameter('contao.image.valid_extensions')), 'mandatory'=>true),
```

Comment:
Maybe stupid proposal but because everybody has to do this in their own extensions too. What if we supported parameters in certain options here?

> `'extensions' => '%contao.image.valid_extensions%'

Wouldn't that simplify things a lot. If this file were in YAML and not a PHP file, one couldn't access the container either.

Reply:
Sounds like totally unrelated to this PR? 😬

Reply:
Well, @ausi had to use the container several times here whereas he could've replaced it with a parameter.

Reply:
Like so 60f4e9be2dc45caa5c033349500c3167e904ed62 ?
Which other options do we want to support?

Reply:
This is a very good idea! I like it. 👍🏻

Reply:
Yeah, that's exactly what I wanted :) Just move it to it's own `protected` method so that any widget can define which properties should be supported for that :)